include/gz/math/MassMatrix3.hh

```diff
@@ -930,6 +930,82 @@ namespace gz
         return this->SetMoi(R * L * R.Transposed());
       }
 
+      /// \brief Set inertial properties based on a Material and equivalent
+      /// cone aligned with Z axis.
+      /// \param[in] _mat Material that specifies a density. Uniform density
+      /// is used.
+      /// \param[in] _length Length of cone along Z axis.
+      /// \param[in] _radius Radius of cone.
+      /// \param[in] _rot Rotational offset of equivalent cone.
+      /// \return True if inertial properties were set successfully.
+      public: bool SetFromConeZ(const Material &_mat,
+                                    const T _length,
+                                    const T _radius,
+                            const Quaternion<T> &_rot = Quaternion<T>::Identity)
+      {
+        // Check that density, _radius and _length are strictly positive
+        // and that quaternion is valid
+        if (_mat.Density() <= 0 || _length <= 0 || _radius <= 0 ||
+            _rot == Quaternion<T>::Zero)
+        {
+          return false;
+        }
+        T volume = GZ_PI * _radius * _radius * _length;
+        return this->SetFromConeZ(_mat.Density() * volume,
+                                      _length, _radius, _rot);
+      }
+
+      /// \brief Set inertial properties based on mass and equivalent cone
+      /// aligned with Z axis.
+      /// \param[in] _mass Mass to set.
+      /// \param[in] _length Length of cone along Z axis.
+      /// \param[in] _radius Radius of cone.
+      /// \param[in] _rot Rotational offset of equivalent cone.
+      /// \return True if inertial properties were set successfully.
+      public: bool SetFromConeZ(const T _mass,
+                                    const T _length,
+                                    const T _radius,
+                            const Quaternion<T> &_rot = Quaternion<T>::Identity)
+      {
+        // Check that _mass, _radius and _length are strictly positive
+        // and that quaternion is valid
+        if (_mass <= 0 || _length <= 0 || _radius <= 0 ||
+            _rot == Quaternion<T>::Zero)
+        {
+          return false;
+        }
+        this->SetMass(_mass);
+        return this->SetFromConeZ(_length, _radius, _rot);
+      }
+
+      /// \brief Set inertial properties based on equivalent cone
+      /// aligned with Z axis using the current mass value.
+      /// \param[in] _length Length of cone along Z axis.
+      /// \param[in] _radius Radius of cone.
+      /// \param[in] _rot Rotational offset of equivalent cone.
+      /// \return True if inertial properties were set successfully.
+      public: bool SetFromConeZ(const T _length,
+                                    const T _radius,
+                                    const Quaternion<T> &_rot)
```

Comment:
indentation

Reply:
I copied this indentation style from SetFromCylinderZ to keep it under 80 char len:
https://github.com/gazebosim/gz-math/blob/118af0968fcb131338f9ebb9bba881ce43dbaa96/include/gz/math/MassMatrix3.hh#L1017-L1044

Reply:
Please re-open these associated requests if they (Cylinder and Cone) both need changed.